ConfAnalyser/cyclohexane.cpp

```diff
@@ -99,17 +100,17 @@ bool Cyclohexane::initialize(const vector<Atom*> &atoms)
 
 bool Cyclohexane::is_flat() const
 {
-        /* flat conforamtion has all atoms in one plane */
+        /* flat conformation has all atoms in one plane */
         if (!has_plane) {
                 return false;
         }
 
         Plane_3D left_plane(*(C[begin]), *(C[(begin+1)%6]), *(C[(begin+4)%6]));
         Plane_3D right_plane(*(C[begin]), *(C[(begin+1)%6]), *(C[(begin+3)%6]));
         return left_plane.is_on_plane(*(C[(begin+2)%6]), tolerance_flat_in) &&
-               left_plane.is_on_plane(*(C[(begin+5)%6]), tolerance_flat_in) &&
+               left_plane.is_on_plane(*(C[(begin+5)%6]), tolerance_flat_in) &&  // are atoms 2 and 5 on the left plane?
```

Comment:
Please, remove this comment, the principles of "how does this work" should be clear from from documentation you are preparing in another branch.